### PR TITLE
Add Logitech G HUB.app

### PR DESCRIPTION
**Pull request by tangestani**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask-drivers/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-drivers/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/logitech-g-hub.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+cask 'logitech-g-hub' do
+  version :latest
+  sha256 :no_check
+
+  # logi.com was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/logitech-g-hub.rb:5]: `download01.logi.com`

**Comment by vitorgalvao** [comment]
Needs to be submitted to [homebrew/cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers). We [define drivers](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) as:

> Software to make a hardware peripheral recognisable and usable by the system. If the software is useless without the peripheral, it’s considered a driver.

Note that repo has [special naming rules](https://github.com/Homebrew/homebrew-cask-drivers/blob/master/CONTRIBUTING.md#naming-driver-casks).

**Comment by tangestani** [comment]
@vitorgalvao I'm confused... Didn't I submit the pull request to [homebrew/cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers)? Please let me know if I'm missing something.

Also, the cask already seems to fit the repo's [special naming rules](https://github.com/Homebrew/homebrew-cask-drivers/blob/master/CONTRIBUTING.md#naming-driver-casks).

**Comment by vitorgalvao** [comment]
> I'm confused... Didn't I submit the pull request to [homebrew/cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers)?

You did. I was the one confused, and though we were on a different repo.

Seems you made some subsequent changes to the branch, stopping me from reopening the PR. Please resubmit.

**Comment by tangestani** [comment]
Yeah I changed the hostname in the comment like you suggested.

Resubmitted the PR: #1655

**Pull request closed without merging**